app/src/main/java/com/goayo/debtify/model/Group.java

```diff
@@ -1,7 +1,10 @@
 package com.goayo.debtify.model;
 
+import androidx.databinding.DataBindingUtil;
```

Comment:
this shouldn't be here

Reply:
It's goneified!

Reply:
Begone unnecessary imports! Begone! The power of Intellij compels you!! 

*holds "Design Patterns - by Gang of Four" up while starting to levitate